core/src/main/java/com/linecorp/armeria/annotations/PathVariable.java

```diff
@@ -0,0 +1,18 @@
+package com.linecorp.armeria.annotations;
```

Comment:
Given that these annotations are only applicable to the strategies given to `HttpDynamicService`, how about moving them to the same package with `HttpDynamicService`?